test/unit/user.server.test.ts

```diff
@@ -4,9 +4,55 @@ import {
   deactivateUserByUsername,
   deleteUserByUsername,
   isDeactivated,
+  getUserByUsername,
+  isStudent,
+  isAdmin,
+  isFaculty,
 } from '~/models/user.server';
 
 import type { User } from '@prisma/client';
+import { prisma } from '~/db.server';
+import { buildUserBaseDomain } from '~/utils';
+
+describe('createUser()', () => {
+  let user: User;
+
+  beforeAll(async () => {
+    user = await createUser(
+      'jsmith',
+      'John Smith',
+      'jsmith@myseneca.ca',
+      'mycustomdomain-students'
+    );
+  });
+
+  afterAll(async () => {
+    await prisma.user.deleteMany().catch(() => {});
```

Comment:
Could use `deleteUserByUsername` instead similar to the rest of the file, or is there a reason you went with this approach?

Reply:
I prefer this as a catch-all way of cleaning all users. If anyone ever create more than one user as part of a test suite, like in the last suite starting on line 166, it can handle the clean-up without having to be modified.
Using `deleteUserByUsername()` means you have to specifically delete that one user, and it requires test writers to keep track of things.

Reply:
Plus, I'm leaning toward not even seeding the test database, to keep the state of the test database "clean". Another reason I want to clean up all users.
But that's probably another discussion/issue.

Reply:
Thank you!